testing/firefox/fix-rust-target.patch

```diff
@@ -1,10 +1,12 @@
 --- a/build/moz.configure/rust.configure
 +++ b/build/moz.configure/rust.configure
-@@ -280,9 +280,11 @@
+@@ -280,9 +280,13 @@
              die("Don't know how to translate {} for rustc".format(
                  host_or_target.alias))
  
-+        fixed_alias = rustc_target.alias.replace("-unknown-", "-alpine-").replace("-gnu", "-musl")
++        fixed_alias = rustc_target.alias.replace("-gnu", "-musl")
++        if host_or_target.cpu != 'aarch64':
++            fixed_alias = fixed_alias.replace("-unknown-", "-alpine-")
 +
```

Comment:
Maybe setting `RUST_TARGET` appropriately would work too? Something like:

`RUST_TARGET=${CTARGET/alpine/unknown}`

Reply:
That looks promising indeed, I think we'll be able to get rid of that patch altogether. I'll wait for the build to finish and post second revision.

Any idea of something to test to detect if we have rust for alpine-linux or unknown-linux in the build environment? That would probably be better than hardcoding it in the APKBUILD depending on the arch.

Reply:
Right now it's unknown-linux on everything but x86_64. Please just hardcode it for now :)

Reply:
You can find the triplets in ./community/rust/APKBUILD

Reply:
Updated version seems to work on aarch64 without `fix-rust-target.patch`, let me know if you can test it on x86_64.

Reply:
Just tested, sadly it doesn't work, sorry about that. I guess we'll have to keep using the patch.

(Also, how is Mozilla of all people not following the standard of RUST_TARGET...)

Reply:
With that reversed it works for me on x86_64

Reply:
Alright thanks for testing, I reverted to patching `rust.configure`. I'll try to debug this and make `RUST_CONFIGURE` work when I have a x86_64 setup.